src/Core/Grid/Definition/Factory/Security/Session/CustomerGridDefinitionFactory.php

```diff
@@ -273,7 +282,17 @@ protected function getBulkActions(): BulkActionCollectionInterface
     {
         return (new BulkActionCollection())
             ->add(
-                $this->buildBulkDeleteAction('admin_security_sessions_customer_bulk_delete')
+                (new SubmitBulkAction('delete_selection'))
+                    ->setName($this->trans('Delete selected', [], 'Admin.Actions'))
```

Comment:
Where does this string appear? 

The delete button should be "Delete" only. And the title should be "Delete session?" (singular) or "Delete selected sessions?" (plural) so I don't understand what this string is.

Reply:
![image](https://user-images.githubusercontent.com/1533248/155357685-b145bfe1-0be4-4c48-b1ba-a88437302b9c.png)

The wording is in french "Supprimer la sélection". I don't think it should be changed as it's common to all grids.

Reply:
Ok, thanks I thought it was the title of the modal.